test/spec/function-references/call_ref.txt

```diff
@@ -0,0 +1,19 @@
+;;; TOOL: run-interp-spec
```

Comment:
It turned out that `call_ref.wast` is already in the repository, and it is passing now. There are other tests in the directory (e.g. `binary.wast`) which is also passing now, they could be updated later as well.